### ISSUE TITLE
TypeError: sequence item 1: expected string, NoneType found

### ISSUE THREAD

**Issue by lgsaber** [maxpumperla/hyperas#85]
I run into this error after pip install --upgrade to the newest master version of hyperas when running the code below:
```
from __future__ import print_function
import theano
import numpy as np,sys,h5py,cPickle,argparse,subprocess
from hyperopt import Trials, STATUS_OK, tpe
from hyperas import optim
from os import system
from os import makedirs
from os.path import join,dirname,basename,exists
from keras.models import model_from_json
...
MAX_EVAL = 30
best_run, best_model = optim.minimize(model=mymodel.model,data=mymodel.data,algo=tpe.suggest,max_evals=MAX_EVAL,trials=Trials())
```
Complete error message:
``` 
 File "main.py", line 56, in <module>
    best_run, best_model = optim.minimize(model=mymodel.model,data=mymodel.data,algo=tpe.suggest,max_evals=MAX_EVAL,trials=Trials())
File "/cluster/geliu/miniconda2/lib/python2.7/site-packages/hyperas/optim.py", line 43, in minimize
    notebook_name=notebook_name, verbose=verbose)
  File "/cluster/geliu/miniconda2/lib/python2.7/site-packages/hyperas/optim.py", line 63, in base_minimizer
    model_str = get_hyperopt_model_string(model, data,functions,notebook_name, verbose, stack)
  File "/cluster/geliu/miniconda2/lib/python2.7/site-packages/hyperas/optim.py", line 130, in get_hyperopt_model_string
    imports = extract_imports(cleaned_source, verbose)
  File "/cluster/geliu/miniconda2/lib/python2.7/site-packages/hyperas/utils.py", line 44, in extract_imports
    import_parser.visit(tree)
  File "/cluster/geliu/miniconda2/lib/python2.7/ast.py", line 241, in visit
    return visitor(node)
  File "/cluster/geliu/miniconda2/lib/python2.7/ast.py", line 249, in generic_visit
    self.visit(item)
  File "/cluster/geliu/miniconda2/lib/python2.7/ast.py", line 241, in visit
    return visitor(node)
  File "/cluster/geliu/miniconda2/lib/python2.7/site-packages/hyperas/utils.py", line 14, in visit_Import
    if (self._import_asnames(node.names)!=''):
  File "/cluster/geliu/miniconda2/lib/python2.7/site-packages/hyperas/utils.py", line 36, in _import_asnames
    return ''.join(asname)
TypeError: sequence item 1: expected string, NoneType found
```

**Comment by simeneide** [comment]
This issue seems to be faced by several others in the (closed) thread #27.

**Comment by jrowen** [comment]
I think this error is coming from `utils._import_asnames` and occurs because one of the `import` statements is not using `as` (e.g. `import theano` instead of `import theano as th`).

Making the following update in `utils` seems to fix the error in my case, but I haven't done much testing.
```
def _import_asnames(self, names):
        asname=map(attrgetter('asname'), names)
        return ''.join(filter(None, asname))
```
